hugo/data/home/home.yml

```diff
@@ -1,37 +1,39 @@
-hero:
-  title: Building the technological stack for AI on code
-  desc: source{d} is building the open-source components that enable machine learning on source code. We believe that the next step in programming comes from teaching machines to understand and write code.
+title: "building the technological stack for AI on code"
+desc: "source{d} is building the open-source components that enable machine learning on source code. We believe that the next step in programming comes from teaching machines to understand and write code."
+links:
+  roadmap:
+    url: http://blog.sourced.tech/post/our-roadmap
+    text: our roadmap
+  manifesto:
+    url: http://sourced.tech/manifesto
+    text: our manifesto
 
-  links:
-    - text: roadmap
-      url: http://blog.sourced.tech/post/our-roadmap
-      newTab: false
-    - text: manifesto
-      url: /manifesto
-      newTab: false
+slack:
+  title: "join the comminity"
+  desc: "we'd be thrilled to have you on our slack"
```

Comment:
Need to check because there is an inconsistency in documentation regarding the Slack world.